### PR TITLE
Update product columns list grid

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request updates product columns list grid

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- yes

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/products.html

### CODE REVIEW

File: src/catalog/products.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ _Products Grid_
 |SKU|The unique Stock Keeping Unit that is assigned to the product.|
 |Price|The unit price of the product.|
 |Quantity|The quantity that is currently in stock.|
+|Salable Quantity|The quantity already sold.|
```
Comment (leslietilling) [inline src/catalog/products.md:43]: I believe this is the quantity that is available for sale.

Comment (ZhovnirMaria) [reply src/catalog/products.md]: @leslietilling your suggestion really makes sense, thank you ☺️

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**